.github/workflows/setup_and_typescript_tests.yml

```diff
@@ -0,0 +1,51 @@
+name: Setup and typescript tests
+on:
+  push:
+    branches:
+      - develop
+      - release-*
+  pull_request:
+    branches:
+      - develop
+      - release-*
+
+jobs:
+  setup_and_typescript_tests:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.7'
+          architecture: 'x64'
+      - uses: ./.github/actions/merge
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: /home/runner/work/oppia/yarn_cache
+          key: ${{ runner.os }}-build-${{ env.cache-name }}-${{ hashFiles('yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-build-${{ env.cache-name }}-
+            ${{ runner.os }}-build-
+            ${{ runner.os }}-
+      - uses: ./.github/actions/install-oppia-dependencies
+      - id: check-risk
+        name: Check if PR is low-risk
+        uses: ./.github/actions/check-if-pr-is-low-risk
+      - name: Install chrome
+        if: ${{ steps.check-risk.outputs.is-low-risk != 0 }}
+        run: python -m scripts.install_chrome_for_ci
+      - name: Check that all e2e test files are captured in protractor.conf.js
+        run: python -m scripts.check_e2e_tests_are_captured_in_ci
```

Comment:
Is his expected to e a part of this workflow?

Reply:
Earlier it was a part of the Typescript test on  Circle Ci. Hence, added it that time. But now looking at it carefully, I don't think so this is related to the typescript tests. Hence, removed.

Reply:
Do we perform this check anywhere else? If not then we have to create a new workflow for it or maybe add it in another relevant workflow?

Reply:
I think a separate workflow will help make things isolate and one won't have to go through the log to check where its e2_coverage or typescript tests actually failing.

But I'm also fine with `e2e_coverage_and_typescript_tests.yml` considering a separate workflow will need separate containers and the e2e test coverage and typescript test can be run together as they take very less time. Feel free to change the name for the workflow.

Reply:
Done